README.md

```diff
@@ -26,7 +26,9 @@ Many models of Orange Pi hardware running [Armbian](https://www.armbian.com/) ar
 
 1. Plug the Pi into your router, so it has connectivity to the Internet. SSH into the Pi with `ssh pi@raspberrypi.local` and password **raspberry**.
 
-    **Optional:** There are other ways to connect, such as connecting the Pi to your computer and sharing Internet to it. Or if you have multiple Pi's connected to your router, find its IP with `nmap -sn 192.168.X.0/24` (where 192.168.X is your subnet) and SSH to the local IP assigned to the Pi you want to address `ssh pi@192.168.X.Y`.
+    **Optional:** There are other ways to connect, such as connecting the Pi to your computer and sharing Internet to it. Or if you have multiple Pi's connected to your router, find its IP with `nmap -sn 192.168.X.0/24` (where 192.168.X is your subnet) and SSH to the local IP assigned to the Pi you want to address `ssh pi@192.168.X.Y`.  
+
+    **Note:** After the install the node will be renamed to tomesh-xxxx where xxxx is the last 4 characters of your CJDNS address.  Before the reboot the node will notify you of what the name is.
```

Comment:
* In code blocks:
  * `tomesh-xxxx`
  * `xxxx`
* cjdns is written as lower case elsewhere in this repo
* Single space after periods